tests/authorities/helpers.py

```diff
@@ -60,6 +62,13 @@ def trans_prep(source, xml_part_to_add):
             verbose=False,
             transform=False
         )
+    elif source == 'idref':
```

Comment:
I think switch case is better for multi way branching

Reply:
There is no direct switch case in python, maybe the following dictionary could help.